### ISSUE TITLE
Help command and flag have a name conflict

### ISSUE THREAD

**Issue by cdlm** [cdlm/clap-st#27]
This requires some boilerplate in the root command (or any that wants to support both `foo help` and `foo --help`)

**Comment by capsulecorplab** [comment]
If the issue is concerning what the default behavior should be, https://clig.dev/#help suggests displaying help text when passed no options, the `-h` flag, or the `--help` flag.

**Comment by cdlm** [comment]
Not really: commands and flags are looked up by their base name which doesn't allow to differentiate between commands and flags. So in practice you get the first one found in the internal data structure, which is not great.

A fix should not be too hard to implement, it's mostly a matter of UX and keeping the code clean. This might already been fixed actually, I haven't had a look at the code in a while and don't remember where I left things.

**Comment by capsulecorplab** [comment]
Is this something that could be potentially resolved with a unit test package (spec'd in a baseline)?

**Comment by cdlm** [comment]
This issue is probably fixed, given [this test](https://github.com/cdlm/clap-st/blob/3f31f13c6861db4cc0c8030c9326a14af96de588/src/Clap-Tests/ClapParameterizedTest.class.st#L27) exists (rejecting duplicate identifiers).
